CGAL_ImageIO/src/CGAL_ImageIO/gis_impl.h

```diff
@@ -143,7 +143,7 @@ int writeGis( char *name, _image* im) {
 		if ( j<n && i<size ) sprintf( str+strlen(str), " " );
 	      }
 	      sprintf( str+strlen(str), "\n" );
-	      res = ImageIO_write( im, str, strlen( str )  );
+	      res = static_cast<int>(ImageIO_write( im, str, strlen( str )));
 	      if ( res  <= 0 ) {
```

Comment:
This code is completely wrong! The error condition should be `res < strlen(str)` and not `res <=0`.

I know that it is not your fault, @cjamin.

To improve the code, I would:
  - use `std::size_t` for the type of `res`
  - and only use the static cast at the very end of the function: `return static_cast<int>(res);`.